content/pages/addons.html

```diff
@@ -0,0 +1,264 @@
+        and **reduce support requests** from old version of your product.
+        {% endmarkdown %}
+      {% endcall %}
+
+    </div>
+  </div>
+
+  {{ about_divide() }}
+
+  <div class="ui very padded container" id="search">
+    <div class="ui vertically padded centered grid">
+
+      {% call about.step(
+          header=header_3,
+          image="/images/docs-as-code/search.png",
+          image_alt="Read the Docs pull request review checks in a GitHub pull request",
+          icon=icon_3) %}
+        {% markdown %}
+        Always find what you are looking for at the tip of your fingers with our powerful ElasticSearch engine.
+        Search on a **single or accross multiple projects** at once.
+        {% endmarkdown %}
+      {% endcall %}
+
+    </div>
+  </div>
+
+  {{ about_divide() }}
+
+  <div class="ui very padded container" id="visual-diff">
+    <div class="ui vertically padded centered grid">
+
+      {% call about.step(
+          header=header_4,
+          image="/images/docs-as-code/visual-diff.png",
+          image_alt="Visual diff between two versions",
+          icon=icon_4) %}
+      {% markdown %}
+      Visuallize documentation changes *"in situ"* from pull requests before merging them.
+      **Understand what exacly changed** in a simple and visual way.
+      {% endmarkdown %}
+      {% endcall %}
+
+    </div>
+  </div>
+
+  {{ about_divide() }}
+
+  <div class="ui very padded container" id="flyout">
+    <div class="ui vertically padded centered grid">
+
+      {% call about.step(
+          header=header_5,
+          image="/images/docs-as-code/flyout.png",
+          image_alt="Read the Docs flyout",
+          icon=icon_5) %}
+      {% markdown %}
+      Let your users select between **multiple versions and translations** of your documentation in a quick way using the flyout menu.
+      {% endmarkdown %}
+      {% endcall %}
+
+    </div>
+  </div>
+</section>
+
+
+{# Try it out code block #}
+{% include "includes/try-it-out-docusaurus.html" %}
+
+{% block homepage_bottom_callout %}
+  <section>
+    <div class="ui very padded container">
+      <div class="ui grid center aligned">
+        <div class="row">
+          <div class="column twelve wide computer sixteen wide tablet">
+            <div class="ui basic vertical huge segment">
+              <h2 class="ui center aligned header">
+                Still not convinced? Try it yourself  <i class="fad fa-rocket icon"></i>
+              </h2>
+
+                {% markdown %}
+                Import your existing project or create one following our [tutorial](https://docs.readthedocs.io/en/stable/tutorial/),
+                and try it yourself.
+                All these features will be enabled by default when using the
+                [``build.commands``](https://docs.readthedocs.io/en/latest/config-file/v2.html#build-commands)
+                config key as shown in the example.
```

Comment:
We should eventually link to the blog post I'm writing at https://github.com/readthedocs/website/pull/278 that says how to enable addons properly.